### ISSUE TITLE
query regarding the channel layer

### ISSUE THREAD

**Issue by devraj89** [udibr/noisy_labels#4]
Hi

Thanks for the wonderful code. I looked into your code and have run the Keras program and was able to replicate your results. However, I have a doubt regarding the channel layer. I will be extremely grateful if you clear this doubt.

In the channel layer implementation in the file `channel.py`
[1] Line `L43`, `channel_matrix = self.activation(self.kernel)`,  are the `softmax` activation applied across the `channel_matrix` `row-wise` or `column-wise` ?
Can you please clarify selecting one over the other? I thought since we are using the `confusion matrix` as the initialization step, it should be row stochastic?

[2] Line `L57`, is the dot product of the 'baseline model' outputs done over the 'rows' or 'columns' of the channel matrix?

Thanks in advance! 
Devraj
